src/Volume.ts

```diff
@@ -260,6 +272,7 @@ export default class Volume {
   /** Returns `true` iff differences between `loadSpec` and `loadSpecRequired` indicate new data *must* be loaded. */
   private mustLoadNewData(): boolean {
     return (
+      this.loadSpec.useExplicitLevel !== this.loadSpecRequired.useExplicitLevel || // explicit vs automatic level changed
       this.loadSpec.time !== this.loadSpecRequired.time || // time point changed
```

Comment:
nit but these could be constants so the final line reads something like `return explicitVsAutoLevelChanged || timePointChanged` so you don't need the comments